### DISCUSSION TITLE
edgemesh-server chart missing in release 1.12?

### DISCUSSION THREAD

**Discussion by ageoffron** [kubeedge/kubeedge#4309] [General]
Noted that the edgemesh server helm chart is missing. from release 1.12
here --> https://github.com/kubeedge/edgemesh/tree/release-1.12/build/helm/edgemesh/charts
Noted that the edgemesh server helm chart is present  in release 1.11
here --> https://github.com/kubeedge/edgemesh/tree/release-1.11/build/helm/edgemesh/charts

Additionally, docker image tag 1.12 is missing.. while agent tag 1.12 is present.
here --> https://hub.docker.com/r/kubeedge/edgemesh-server/tags

Yet the edgemesh documentation refer to edgemesh 1.12 overall...
"After EdgeMesh v1.12.0, the capabilities of edgemesh-server are merged into the tunnel module of edgemesh-agent, so that edgemesh-agent with relay capability can automatically become a relay server, providing other nodes with the functions of assisting hole punching and relaying"

**Comment by vincentgoat** [comment]
cc @Poorunga

**Comment by ageoffron** [comment]
commenting back for anyone else. 
From edgemesh 1.11 to 1.12 edgemesh server is not a thing anymore.
All edgemesh agent are deployed and one is to be elected as the server via its "relaying capabilities".
